kowl/templates/deployment.yaml

```diff
@@ -15,6 +15,9 @@ spec:
     metadata:
       annotations:
         checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+        {{- if .Values.podAnnotations }}
+        {{ toYaml .Values.podAnnotations | indent 8 }}
```

Comment:
@weeco done!

Reply:
We need to use nindent instead of indent

Reply:
oops, sorry, fixed